extension.js

```diff
@@ -169,12 +171,21 @@ function assertNextJSApp(componentPath) {
  */
 async function executeCommand(commandInput, componentPath) {
 	const [command, ...args] = shellQuote.parse(commandInput);
-	const cp = child_process.spawn(command, args, {
+	const env = { ...process.env, PATH: `${process.env.PATH}:${componentPath}/node_modules/.bin` };
```

Comment:
I thought with an `exec`, that the pathing to the node_modules/.bin directory, but maybe I am misremembering. If this works, that's fine.

Reply:
I thought so too, but I believe that is strictly when npm is being invoked. Since the default build command is just `next build` we need to set the bin here. 

Im considering changing the default to `npm run build` but there's some UX to consider there. I'll review tomorrow

Reply:
Okay so the reason why I can't have the default be `npm run build`, is that the ideal UX is that the user changes their build script to: `harperdb-nextjs build`. Thus, `{ "scripts": { "build": "harperdb-nextjs build" } }` in their package.json. This enables the user to locally run `npm run build` and 🪄 . If the extension would only ever run `npm run build`, then this would create a infinite loop.